### ISSUE TITLE
bandprotocol makes incorrect prices in price-server

### ISSUE THREAD

**Issue by BlockNgine** [terra-project/oracle-feeder#29]
When I upgraded to v1.3.6 on testnet i also enabled bandprotocol in default.js; but with it I'm getting incorrect data, and correct data when I disable it:

example incorrect:

```
2021-04-27 06:45:01 {
  'LUNA/KRW': '22347.451578257363739914',
  'LUNA/USD': '19.002181157367612074',
  'LUNA/SDR': '13.232795098747916289',
  'LUNA/EUR': '15.743367977555863791',
  'LUNA/CNY': '123.159456211166038263',
  'LUNA/JPY': '2056.520235328457914045',
  'LUNA/GBP': '18.220404756483933870',
  'LUNA/INR': '1886.963519264784366198',
  'LUNA/CAD': '31.386411086662971666',
  'LUNA/CHF': '23.171096448977245894',
  'LUNA/HKD': '196.480668559078223243',
  'LUNA/AUD': '32.468549215344431319',
  'LUNA/SGD': '33.552842990710355117',
  'LUNA/THB': '794.276038392507705176',
  'LUNA/MNT': '72324.248251227011408755',
  'USDT/KRW': '1177.357864138695503910',
  'BTC Premium': '294.805118507100874848'
}
```

USD price is correct
but GBP/EUR and  BTC Premium; the examples i'm looking at are incorrect
I'm going to hold off enabling it on Mainnet for now
Is anyone else experiencing this?

**Comment by hanjukim** [comment]
Please hold on. It is wrong.

**Comment by hanjukim** [comment]
It is fixed in #30 and release has been deployed: https://github.com/terra-project/oracle-feeder/releases

**Comment by BlockNgine** [comment]
Thanks!
